Kernel/Output/HTML/Templates/Standard/AgentPreferences.tt

```diff
@@ -209,9 +209,6 @@
 
                                 <div class="SettingDescription">
                                     [% Translate(Data.Desc) | html %]
-                                [% IF Data.Label == 'Language' %]
-                                    [% Translate("Did you know? You can help translating OTRS at %s.") | html | ReplacePlaceholders('<a href="https://translate.otrs.com" target="_blank">translate.otrs.com</a>') %]
-                                [% END %]
```

Comment:
Can you please undo the change.
The change does not fit thematically with the others (subject). 
Keep it simple. You are welcome to use another pull request.

Reply:
**We will not merge pull requests whose code change includes multiple topics!
Changing the title is not a solution, it is not acceptable! **
Please create a new pull request for this.